documentation/asciidoc/topics/proc_connecting_cli.adoc

```diff
@@ -27,12 +34,25 @@ $ {kubectl_exec}
 However using the CLI in this way consumes memory allocated to the container, which can lead to out of memory exceptions.
 ====
 
+.Procedure
+
 . Create a CLI connection to your {brandname} cluster.
 +
+ifdef::downstream[]
+.Using the server distribution
+endif::downstream[]
 [source,bash,options="nowrap",subs=attributes+]
 ----
 $ bin/cli.sh -c https://$SERVICE_HOSTNAME:$PORT --trustall
 ----
+ifdef::downstream[]
++
+.Using the native CLI
+[source,bash,options="nowrap",subs=attributes+]
+----
+$ ./redhat-datagrid-cli -c https://$SERVICE_HOSTNAME:$PORT
```

Comment:
I guess you can add `--trustall` so the commands (native vs non-native) are identical